study-builder/fdahpStudyDesigner/src/main/webapp/WEB-INF/view/notification/createOrUpdateNotification.jsp

```diff
@@ -438,7 +438,7 @@
         $('#timepicker1').val('');
         $('.timepicker').parent().addClass('has-error has-danger').find('.help-block.with-errors')
             .empty().append(
-            	$("<ul><li> </li></ul>").attr("class","list-unstyled").text("Please select a time that has not already passed for the current date"));
+            	$("<ul><li> </li></ul>").attr("class","list-unstyled").text("Please select a time that has not gone past yet"));
```

Comment:
This wording sounds incorrect. Please use "Please select a time in the future."

Reply:
Fixed review comment